### PR TITLE
added ReadMe

### LINKED ISSUE #3: Update ReadMe
Update ReadMe to fit documentation

### PR DESCRIPTION
**Pull request by Joven-Heng**
close #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,22 @@
...
+* At the end of the project, we would want to ensure that: 
+  * Our end product is a **working product** and can actually be used to record client information. 
+  * Our end product will have **few/no** bugs. 
+  * The project comes with a **reasonable number of tests**.
   * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+* It is named `InsuranceForInsurance` (`I4I` for short) because we thought that this app would be a form of insurance for insurance advisors such that they do not forget their clients.
+* For the detailed documentation of this project, see **insert website here**
```
Comment (ijavierja) [inline README.md:19]: No website inserted yet

**Review by ijavierja — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (WeiJie96) [inline README.md:19]: Perhaps it would be good if we insert our website here.

**Review by WeiJie96 — CHANGES_REQUESTED** [review]
Overall is good, just that it would be good to insert our website.

**Review by WeiJie96 — APPROVED** [review]
LGTM

**Review by ijavierja — APPROVED** [review]
LGTM - merge

**Pull request merged**